### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by ghost** [dkundel/twilio-conference-dashboard#3]
I assume the reason this isn't working is because it's outdated. Is there a chance you could update it? It looks awesome!

**Comment by dkundel** [comment]
Yeah I haven't had time to work on this for a while. Can you tell me what's broken and I can try to see if I can put some time into fixing it.

Cheers,
Dominik

**Comment by PLKids** [comment]
I think it’s just outdated. Thanks!
